File: src-mac/Sources/K3UsbSafeMac/K3Version.swift
import Foundation

enum K3Version {
    static let appVersion = "0.4.0-mac"
    static let buildDate = "2026-07-10"
    static let channel = "portable"

    static var display: String {
        "\(appVersion) (\(channel), \(buildDate))"
    }

    static let changelog = [
        "First-run wizard",
        "Folder encryption with structure preservation",
        "USB integrity manifest",
        "K3 rule updater",
        "Quarantine detail panel",
        "Drag and drop encrypt/decrypt",
        "macOS clone warning"
    ]
}

File: src-csharp/K3Version.cs
using System;

namespace AnToanUSB
{
    public static class K3Version
    {
        public const string AppVersion = "0.4.1-win";
        public const string BuildDate = "2026-07-11";
        public const string Channel = "portable";

        public static string Display
        {
            get { return AppVersion + " (" + Channel + ", " + BuildDate + ")"; }
        }

        public static readonly string[] Changelog = new string[]
        {
            "Windows first-run wizard and login lockout policy",
            "Folder encryption with structure preservation",
            "USB integrity manifest",
            "K3 rule updater from URL/file",
            "HTML scan report export",
            "Quarantine trust and restore",
            "Auto scan on login"
        };
    }
}
